KatyaM/test_task/Server/mock/mock.go

```diff
@@ -0,0 +1,99 @@
+package mock
+
+import (
+	context "context"
+	gomock "github.com/golang/mock/gomock"
+	"reflect"
+	b "test_task/Server/database_interface"
+)
+
+// MockService is a mock of Service interface
+type MockService struct {
+	ctrl     *gomock.Controller
+	recorder *MockServiceMockRecorder
+}
+
+// MockServiceMockRecorder is the mock recorder for MockService
+type MockServiceMockRecorder struct {
+	mock *MockService
+}
+
+// NewMockService creates a new mock instance
+func NewMockService(ctrl *gomock.Controller) *MockService {
+	mock := &MockService{ctrl: ctrl}
+	mock.recorder = &MockServiceMockRecorder{mock}
+	return mock
+}
+
+// EXPECT returns an object that allows the caller to indicate expected use
+func (m *MockService) EXPECT() *MockServiceMockRecorder {
```

Comment:
Do not use upper case
```suggestion
func (m *MockService) expect() *MockServiceMockRecorder {
```